components/build/tekton-chains/chains-certs.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: chains-ca-cert
+  namespace: tekton-chains
```

Comment:
nit: I would not define namespace here but in the kustomization.yaml, less code and in theory you can easily move to different namespace.